e2e/utils/TestUtil.ts

```diff
@@ -0,0 +1,29 @@
+import { Page } from '@playwright/test';
+import { ObjectUtil } from './ObjectUtil';
+
+let ID = 0;
+const uuid = () => `test${ID++}`;
+
+export class TestUtil {
+  constructor(readonly page: Page) {}
```

Comment:
What is that allow us to not assign page explicitly to the constructor?

Reply:
Yes, it is an abbreviation

Reply:
can you link it to me? i want to learn what is it.

Reply:
https://dev.to/satansdeer/typescript-constructor-shorthand-3ibd